scripts/opensearch-setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+e=$(curl -o /dev/null -s -w "%{http_code}\n" localhost:9200)
```

Comment:
Add comments here for what we are doing.

Also, this file shouldn't be named opensearch-setup any more, considering that its functionality is to check if opensearch is running. Because it is only 6 lines of code, I think you can just move it into the entrypoint.

Reply:
I've refactored everything into `opensearch-entrypoint.sh` and added more comments to illustrate the process. I've also added a time limit to kill the tool if OpenSearch hasn't started in 2 minutes.